packages/core/core/src/services/document-service/transform/relations/transform/data-ids.ts

```diff
@@ -63,6 +63,24 @@ const getRelationIds = curry(
   }
 );
 
+/**
+ * Ensure set/connect/disconnect syntax for all relations in the data object
+ */
+const standarizeRelations = (data: Record<string, any>, uid: string) => {
```

Comment:
typo?

```suggestion
const standardizeRelations = (data: Record<string, any>, uid: string) => {
```

Reply:
Typo that's been here for ages indeed :)

Reply:
oh I thought you were adding this function 🙃

Reply:
I don't even remember but I'll fix it anyway